src/pe_reports/report_generator.py

```diff
@@ -22,12 +38,28 @@ def load_template():
 
 def export_set(prs):
     """Export PowerPoint report set to output directory."""
-    pptx_out = "Customer_ID_Posture_Exposure.pptx"
-    prs.save(os.path.join(SAVE_PATH, pptx_out))
+    try:
+        pptx_out = "Customer_ID_Posture_Exposure.pptx"
+        prs.save(os.path.join(SAVE_PATH, pptx_out))
+    except Exception:
+        print("No output available.")
     return
 
 
+def generate_reports(data, data_dir, out_dir):
+    """Gather assets to produce reports."""
+    # TODO: build code to connect customer db, encrypt and embed pdf reports.
+
+
 def main():
+    """Set up logging and call the pe_reprots function."""
+    args: Dict[str, str] = docopt.docopt(__doc__, version=__version__)
+
+    # TODO: Add generate_reports func to handle cmd line arguments and function.
```

Comment:
Please [create an issue](https://github.com/cisagov/pe-reports/issues/new?assignees=&labels=&template=issue.md) for this TODO item and update this comment to include a link to that issue.

Reply:
Issue created: https://github.com/cisagov/pe-reports/issues/8
Noted in commit: 7ba36fa